app.rb

```diff
@@ -61,7 +64,7 @@
       end
     end
 
-    pull_request.check_commits_style if redmine_issue_repos.include?(pull_request.repo) && pr_action != 'created'
+    pull_request.check_commits_style(redmine_project_for_repo(pull_request.repo)) if pr_action != 'created'
```

Comment:
I see this mistakenly changes the logic - on a side note, what does the `&& pr_action != 'created'` do? It doesn't seem to have a side-effect.